lms/validation/authentication/_lti.py

```diff
@@ -0,0 +1,56 @@
+import marshmallow
+
+from lms.lti import CLAIM_PREFIX
+from lms.models import LTIUser
+from lms.services import ApplicationInstanceNotFound
+from lms.validation._base import PyramidRequestSchema
+from lms.validation._exceptions import ValidationError
+from lms.validation._lti import LTIAuthParamsSchema
+
+
+class LTI13AuthSchema(LTIAuthParamsSchema, PyramidRequestSchema):
+    """
+    Schema used to validate the LTI1.3 params needed for authentication.
+
+    Using the lti_user method produces an LTIUser based on those parameters.
+    """
+
+    location = "form"
+
+    iss = marshmallow.fields.Str(required=True)
+    aud = marshmallow.fields.Str(required=True)
+    deployment_id = marshmallow.fields.Str(required=True)
+
+    @marshmallow.pre_load
+    def _lti_v3_fields(self, data, **_kwargs):  # pylint:disable=no-self-use
+        data["iss"] = data.v13["iss"]
```

Comment:
This, using `pre_load` is the clearest way I've found to reference `v13`. 

We could use a custom marshmallow field but as this is the only place this needs to happen that would be overkill and not at all easier to read.

Reply:
There's `Pluck`? Not sure if you tried this:

* https://marshmallow.readthedocs.io/en/stable/marshmallow.fields.html?highlight=Pluck#marshmallow.fields.Pluck

This seems to allow you to surface a sub key into a parent object, but... why not just let it be nested? It seems like we've avoided mixing the params in the lower object just to mix them here? I think this doesn't achieve the aim of keeping them separate which is to highlight which version we are using in the code.

Is the most natural thing here to have a `v13` sub-schema? This would mean the code would look like:

```python
request.parsed_params['v13']['iss']
```

Which is more verbose, but also completely explicit.